ci/scripts/dask_config_to_env.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+"""Read a dask config file and print it out in the format `-e ENV=VALUE ENV=VALUE ...`
+This script is a work-around to not being able to upload dask config files to
+`conda env create`.
+"""
+from __future__ import annotations
+
+import sys
+from typing import Iterator
+
+import yaml
+
+
+def main(fname: str) -> None:
```

Comment:
I believe we're now able to ship dask config using the dask.config ctx manager. That might be a better interface than this. is there a reason why we use env vars instead?

Reply:
> I believe we're now able to ship dask config using the dask.config ctx manager.

Just to be clear, change to ship dask config to cluster is not yet deployed, presumably will go out next week.

Reply:
? This is news to me.
Are you saying that when you call `coiled.create_software_environment` it will pick up the local config?
Also, I needed something that could be passed to `coiled env create`. Ideally one would want to pass it a dask config file directly, but I believe there's no such feature?

Reply:
Now I'm confused. The feature is that when you create cluster it ships local config. I don't see any connection between dask config and the software environment... am I missing something?

Reply:
> Just to be clear, change to ship dask config to cluster is not yet deployed, presumably will go out next week.

This is exactly what we need, please ping me when it's generally available

Reply:
> Now I'm confused. The feature is that when you create cluster it ships local config. I don't see any connection between dask config and the software environment... am I missing something?

The script is currently calling `coiled env create -e DASK_DISTRIBUTED_...=VALUE`

Reply:
> The script is currently calling `coiled env create -e DASK_DISTRIBUTED_...=VALUE`

I could be wrong but I think that just applies when creating the software environment and not when you make a cluster using that software environment.

Reply:
It works fine for me?
```python
import coiled
import dask.config
import distributed

!coiled env create --name crusaderky/test_vars --conda AB_environments/AB_baseline.conda.yaml -e DASK_TESTVAR=123
cluster = coiled.Cluster(name="test_vars", n_workers=0, software="crusaderky/test_vars")
client = distributed.Client(cluster)
client.run_on_scheduler(lambda: dask.config.get("testvar"))
```
```
123
```

Reply:
Oh, thanks, good to know!